pkg/cmd/generate/configurations.go

```diff
@@ -15,16 +15,16 @@ const (
 	envFormat        = "env"
 	jsonFormat       = "json"
 	propertiesFormat = "properties"
-	secretFormat     = "secret"
+	configmapFormat  = "configmap"
 )
 
-var configurationTypes = []string{envFormat, jsonFormat, propertiesFormat, secretFormat}
+var configurationTypes = []string{envFormat, jsonFormat, propertiesFormat, configmapFormat}
 
 var (
-	envConfig            = template.Must(template.New(envFormat).Parse(templateEnv))
-	jsonConfig           = template.Must(template.New(jsonFormat).Parse(templateJSON))
-	propertiesConfig     = template.Must(template.New(propertiesFormat).Parse(templateProperties))
-	secretTemplateConfig = template.Must(template.New(secretFormat).Parse(templateSecret))
+	envConfig             = template.Must(template.New(envFormat).Parse(templateEnv))
+	jsonConfig            = template.Must(template.New(jsonFormat).Parse(templateJSON))
+	propertiesConfig      = template.Must(template.New(propertiesFormat).Parse(templateProperties))
+	congMapTemplateConfig = template.Must(template.New(configmapFormat).Parse(templateConfigMap))
```

Comment:
![image](https://user-images.githubusercontent.com/981838/180230433-9c4a5a6e-0966-4ba4-97c6-f21f1fd4a7b1.png) 
;)
```suggestion
	kongMapTemplateConfig = template.Must(template.New(configmapFormat).Parse(templateConfigMap))
```